### ISSUE TITLE
Allow antimicrox to be installed on non-admin computers (without using hacks like runasinvoker)

### ISSUE THREAD

**Issue by ayunami2000** [AntiMicroX/antimicrox#366]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Installer requires administrator privileges to use despite application not needing them.

### Expected Behavior

It works when using `set __COMPAT_LAYER=RunAsInvoker` in a batch file and then running the installer. This means that the installer does *not* need to require administrator privileges to install. I would expect it to install without needing administrator permissions.

### Steps To Reproduce

1. Run the setup file.

### Environment

```markdown
- OS: Microsoft Windows 10 Education 10.0.19043 Build 19043
- AntiMicroX version: 3.2.1
```

### Anything else?

Basically just make it able to be installed without admininstrator and without needing workarounds like `set __COMPAT_LAYER=RunAsInvoker`

**Comment by pktiuk** [comment]
Hi @ayunami2000  thanks for report and possible solution,  
Bot to be honest, I don't know how to apply change you suggested.   
But any PR tackling this will be welcome

**Comment by ayunami2000** [comment]
Perhaps also provide a .zip version?

**Comment by pktiuk** [comment]
There is https://github.com/AntiMicroX/antimicrox/issues/343 for this

**Comment by koga-ninja** [comment]
Portable version on windows does run without administrator approval required.

**Comment by pktiuk** [comment]
I think @koga-ninja is right, this issue can be closed. :)
